web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -109,13 +72,10 @@ $(function(){
                     height: 'auto',
                     margin: 0
                 });
-                $('#fms_pan_zoom').css({ top: '2.75em' });
```

Comment:
The removal of the different top for mobile/desktop means the navigation is appearing too high up on mobile view.